### PR TITLE
1238/GitHub integration followup fixes

### LINKED ISSUE #1238: Github integration follow-ups
# Overview

We're like to add after the initial implementation:

- more reading data tests (not only metadata assertions but also data)
- probably it will be better to have output descriptors in the tests (as a consts?) not in contest (hard to follow)
- recover writing tests on Windows/MacOS (copy `.env.example` in github actions and disable SQL tests using `skipif`)
- docstrings for control's properties - https://framework.frictionlessdata.io/docs/portals/github.html#reference-portals.githubcontrol
- support catalog from a repository (user request)
- a small blog post that the integration has been added with a link to the docs

### PR DESCRIPTION
**Pull request by shashigharti**
- Catalog can be created from a single repo now.
- new tests and fixed old tests
- more tests for reading data
- docstring for control
- blog post 
- fixes #1238

---

Please make sure that all the checks pass. Please add here any additional information regarding this pull request. It's highly recommended that you link this PR to an issue (please create one if it doesn't exist for this PR)

### CODE REVIEW

**Comment by shashigharti** [comment]
@roll this is ready for review.

**Comment by roll** [comment]
Hi @aivuk,

can you please test and review as you work on similar functionality for CKAN

**Review by aivuk — APPROVED** [review]
LGTM, great work @shashigharti I did some small changes in the documentation, I did not requested because they were really minor ones

**Comment by shashigharti** [comment]
Thanks! @aivuk

**Comment by roll** [comment]
Thanks @shashigharti (and @aivuk for the review). Great job!
